hv.h

```diff
@@ -520,12 +517,7 @@ whether it is valid to call C<HvAUX()>.
 #define Perl_sharepvn(pv, len, hash) HEK_KEY(share_hek(pv, len, hash))
 #define sharepvn(pv, len, hash)	     Perl_sharepvn(pv, len, hash)
 
-#define share_hek_hek(hek)						\
-    (++(((struct shared_he *)(((char *)hek)				\
-                              - STRUCT_OFFSET(struct shared_he,		\
-                                              shared_he_hek)))		\
-        ->shared_he_he.he_valu.hent_refcount),				\
-     hek)
+struct hek * share_hek_hek (const struct hek *);
```

Comment:
I don't like that we now have a global `share_hek_hek` symbol without a `Perl_` prefix.

Reply:
good point, fixed